lib/segment/src/vector_storage/query_scorer/sparse_query_scorer.rs

```diff
@@ -0,0 +1,40 @@
+use common::types::{PointOffsetType, ScoreType};
+use sparse::common::sparse_vector::SparseVector;
+
+use crate::vector_storage::query_scorer::QueryScorer;
+use crate::vector_storage::SparseVectorStorage;
+
+pub struct SparseQueryScorer<'a, TVectorStorage: SparseVectorStorage> {
```

Comment:
ToDo: remove after normal plain search is implemented

Reply:
Added TODO